src/transformers/utils/import_utils.py

```diff
@@ -698,7 +698,7 @@ def is_jumanpp_available():
 
 Note that if you have a local folder named `datasets` or a local python file named `datasets.py` in your current
 working directory, python may try to import this instead of the 🤗 Datasets library. You should rename this folder or
-that python file if that's the case.
+that python file if that's the case. Please note that you may need to restart your runtime after installation.
```

Comment:
```suggestion
that python file if that's the case. Please note that you may need to restart your runtime after installation.
If you're using an ephemeral runtime like Google Colab, you might want to simply perform the installations before any imports.
```

Reply:
I've not added for all occurrences of this text because I'm not sure if this is important enough to be included. I'll add these suggested changes for all occurrences if this is useful!

Reply:
Restarting the runtime won't delete the installations! This will only happen if you select 'disconnect and delete runtime'

Reply:
That option used to be called 'Factory reset runtime' I believe, which might explain some of the confusion.

Reply:
That is correct! I mixed them up, thanks for clearing it up!